RFC-0007-remote-functions.md

```diff
@@ -0,0 +1,219 @@
+# **RFC-0007 for Presto**
+
+## [Title] REST API for remote functions
+
+Proposers
+
+* Tim Meehan
+* Abe Varghese
+* Joe Abraham
+* Jakob Khaliqi
+
+## [Related Issues]
+
+* https://github.com/prestodb/presto/issues/14053
+
+## Summary
+
+### Dynamic functions in remote function servers
+
+A new REST API is defined, along with a REST plugin implementation, which allows for consistent and unified metadata and execution
+of remote functions in a single API definition.  This API is designed to be extensible, allowing for the definition of new functions
```

Comment:
Do you mean dynamic here instead of extensible ? Can we presume that new functions can be defined and used without a server restart ?